trymito.io/pages/spreadsheet-automation.tsx

```diff
@@ -97,6 +97,28 @@ const SpreadsheetAutomation: NextPage = () => {
                 <Image src={'/explore_data_visually.png'} alt='Explore data with Mito' width={500} height={250} layout='responsive'/>
               </div>
             </div>
+            <div className={pageStyles.subsection}>
+              <div className={classNames(textImageSplitStyles.functionality_media, 'display-desktop-only-inline-block')}>
+                <Image src={'/export_to_excel.png'} alt='Mito generated code' width={500} height={250} layout='responsive'/>
+              </div>
+              <div className={textImageSplitStyles.functionality_text}>
+                <h1>
+                  Compatible with <span className='text-highlight'>Excel</span>
+                </h1>
+                <p>
+                  Easily create Excel files with Mito.
```

Comment:
I think this is a good point. But maybe the one missing thing is just "why do I want to generate Excel files with Mito". So maybe we could update this to:

Ideation of these two paragraphs:

p1: Apply conditional formatting, set table colors, and format numbers using Mito's. Don't waste time trying to understand the xlsxwriter API -- it's confusing even to engineers! 
p2: Learn more about generating presentation-ready Excel files.